README.md

```diff
@@ -3,10 +3,26 @@ This repository hosts the Solo.io API definitions. It is intended as a read-only
 APIs are the projects that own them. The goal of this repository is to allow for projects to consume an API 
 without having to depend on the project that owns it.
 
-## Gloo Apis
-Syncing the Gloo APIs is currently a semi-automated process and is done with 2 github actions, requiring manual approval in order to merge into master.
+## Gloo Edge APIs
+For each release of Gloo Edge, we want to freeze the solo-apis dependency by creating a unique tag tied to the dependent release. For example, when Gloo Edge `v1.8.4` is released, a corresponding `v1.8.4` tag is created in the Gloo Edge repository. We want to create a `gloo-v1.8.4` tag in solo-apis with a read-only mirror of the API.
 
-1. In the gloo repo, a github action will run on each Gloo OS release. It will run `./hack/sync-gloo-apis.sh; make generate -B` in solo-apis and push to the `update-gloo-apis` branch in solo-apis.
-2. In the solo-apis repo, each push to `update-gloo-apis` will automatically create a PR.
-3. A member of the solo-io org must approve the pull request.
-4. Once the PR has merged into master, a tag should be created off of master corresponding to the Gloo tag which was used.
+In order to keep API definitions in sync, we automate the process of syncing long term support (LTS) branches. To keep the LTS branches unique, we prefix them with the dependent repo name (ie `gloo`). For example, in gloo the `v1.8.x` LTS branch is tracked by the `gloo-v1.8.x` LTS branch in solo-apis.
+
+### How do Gloo Edge APIs get synced?
+
+1. A Gloo release occurs (ie v1.8.1 is released from LTS branch v1.8.x)
+1. A Gloo GHA generates the new APIs and pushes those to a branch (ie sync-apis/gloo-v1.8.x/gloo-v1.8.1)
+1. A [Solo-Apis GHA](.github/workflows/README.md#create-pr-for-lts-branch) notices that a commit was pushed to a `sync-apis` prefixed branch, and creates a PR.
+1. **That PR is manually reviewed by a developer, and eventually approved**
+1. A [Solo-Apis GHA](.github/workflows/README.md/#tag-commit-on-lts-branch) notices that a commit is pushed to a LTS branch (ie `gloo-v1.8.x`) and tags the commit. This tag is referenced by other repositories.
+
+### A Gloo Edge release occurred, but the Gloo GHA didn't run, how do I manually sync the APIs?
+As noted above, the automated process is kicked of by a Gloo Edge release. In the case where the appropriate GitHub action doesn't run or fails, we need to manually complete what that GitHub action does.
```

Comment:
maybe should mention that the below steps assume you have your local copy of the gloo and solo-apis repos located under the same parent directory